### ISSUE TITLE
Error downloading gnark keys with `./scripts/install.sh` on new Apple M4 Pro

### ISSUE THREAD

**Issue by jacksondoherty** [Lightprotocol/light-protocol#1385]
Error message: 

`curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 1). Retrying...`

Full log:

```
jacksondoherty@Mac light-protocol % ./scripts/install.sh
Installing Go...
Installing Rust...
info: downloading installer
info: profile set to 'default'
info: default host triple is aarch64-apple-darwin
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for 'stable-aarch64-apple-darwin'
info: default toolchain set to 'stable-aarch64-apple-darwin'

  stable-aarch64-apple-darwin unchanged - rustc 1.83.0 (90b35a623 2024-11-26)


Rust is installed now. Great!

To get started you need Cargo's bin directory 
(/Users/jacksondoherty/temporary/garden-labs/light-protocol/.local/cargo/bin) 
in your PATH
environment variable. This has not been done automatically.

To configure your current shell, you need to source
the corresponding env file under 
/Users/jacksondoherty/temporary/garden-labs/light-protocol/.local/cargo.

This is usually done by running one of the following (note the leading DOT):
. "/Users/jacksondoherty/temporary/garden-labs/light-protocol/.local/cargo/env"
# For sh/bash/zsh/ash/dash/pdksh
source 
"/Users/jacksondoherty/temporary/garden-labs/light-protocol/.local/cargo/env.fish"
# For fish
info: component 'clippy' for target 'aarch64-apple-darwin' is up to date
info: component 'rustfmt' for target 'aarch64-apple-darwin' is up to date
    Updating crates.io index
     Ignored package `cargo-expand v1.0.95` is already installed, use --force to override
    Updating crates.io index
     Ignored package `wasm-pack v0.13.1` is already installed, use --force to override
     Summary Successfully installed cargo-expand, wasm-pack!
     Ignored package `photon-indexer v0.50.0` is already installed, use --force to override
Installing Node.js...
Installing pnpm...
Installing Solana...
Installing Anchor...
Installing jq...
Downloading gnark keys...
Warning: Could not determine remote file size for inclusion_26_1.key
Downloading inclusion_26_1.key
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 1). Retrying...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 2). Retrying...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 3). Retrying...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 4). Retrying...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 5). Retrying...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
Download failed for inclusion_26_1.key (attempt 6). Retrying...
```

**Comment by sergeytimoshin** [comment]
@jacksondoherty Thanks for the report! Fixed in #1386.

**Comment by jacksondoherty** [comment]
Thank you!
